guides/plugins/plugins/content/cms/add-cms-element-via-admin-sdk.md

```diff
@@ -0,0 +1,352 @@
+# Add CMS element via Admin Extension SDK
+
+## Overview
+
+This article will teach you how to create a new CMS element via the Admin Extension SDK. The plugin in this example will be named `SwagBasicAppCmsElementExample`, similar to the other guides.
+
+## Prerequisites
+
+You won't learn how to create a plugin or an app in this guide, head over to our Plugin or App Base Guide to create your first extension.
+
+This guide will also not explain how a custom component can be created in general, so head over to the official guide about creating a custom component to learn this first. It also won't go too much into detail about the Admin Extension SDK. For that, please have a look at the [Admin Extension SDK documentation](https://shopware.github.io/admin-extension-sdk/docs/guide/getting-started/installation).
```

Comment:
Replace this info as below

* [Knowledge on the creation of Plugin or App](https://developer.shopware.com/docs/guides/plugins/apps/app-base-guide)
* Knowledge on the creation of [admin custom component](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-component#creating-a-custom-component)
* Understanding on [Admin Extension SDK](https://shopware.github.io/admin-extension-sdk/docs/guide/getting-started/installation)

Reply:
Not 100% sure, if I understood this correctly, but I hope this is fine now :)

Reply:
The bullet points are just an alternate description for the one you had written earlier. Also, in a positive and straightforward way. Request you to replace the two paragraphs just with the bulleted points.

Reply:
Ah, okay. I got that. Pushed :+1: